notebooks/HASP/DataDiagnostic/DataDiagnostics.ipynb

```diff
@@ -0,0 +1,1599 @@
+{
```

Comment:
<p>I suggest a slight tweak: instead of using a list comprehension and <code>np.concatenate</code> about we leverage numpy's <code>reshape</code>  and <code>mean</code>  functions more directly.</p><p> </p><pre class="ql-syntax" spellcheck="false">xs = myarr.shape[0]
crarr = myarr[:xs-(xs % factor)]
dsarr = crarr.reshape(-1, factor).mean(axis=1)
return dsarr
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/207/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>